src/wallet/test/coinselector_tests.cpp

```diff
@@ -74,9 +74,8 @@ static void add_coin(CoinsResult& available_coins, CWallet& wallet, const CAmoun
     tx.vout.resize(nInput + 1);
     tx.vout[nInput].nValue = nValue;
     if (spendable) {
-        auto op_dest = wallet.GetNewDestination(OutputType::BECH32, "");
-        assert(op_dest.HasRes());
-        tx.vout[nInput].scriptPubKey = GetScriptForDestination(op_dest.GetObj());
+        auto op_dest = *(Assert(wallet.GetNewDestination(OutputType::BECH32, "")));
```

Comment:
Same: this also creates a copy without a `std::move`?

If yes, maybe use the same you used in the fuzz test:


`GetScriptForDestination(*Assert(op_dest))`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25721#discussion_r935510002

> Same: this also creates a copy without a `std::move`?

Thanks inlined to avoid copy.